templates/shareaholic-settings.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class='wrap'>
+  <script>
+  window.ShareaholicConfig = {
+    apiKey: "{{ apiKey }}",
+    verificationKey: "{{ verificationKey }}",
+    apiHost: "{{ apiHost }}",
+    serviceHost: "{{ serviceHost }}",
+    assetHost: "{{ assetHost }}",
+    assetFolders: true,
+    origin: "drupal",
+    language: "{{ language }}"
+  };
+  </script>
+  <div id="root" class="shr-site-settings"></div>
+  <script class="shr-app-loader__site-settings" src="https://cdn.shareaholic.net/ui-site-settings/loader.js"></script>
```

Comment:
Can we make the `cdn.shareaholic.net` path a variable (similar to web.shareaholic.com) to make it easier to switch between staging and production code.